setup.sql

```diff
@@ -0,0 +1,149 @@
+CREATE DATABASE novels;
```

Comment:
You can delete this. We only used .sql files until we learned migrations. Most projects don't ever have them because you'd use migrations instead.